index2.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Nawaf Ibrhaim</title>
```

Comment:
you forgot to add   <link rel="stylesheet" type="text/css" href="styles.css"> in your HTML HEAD